src/main/java/com/zufar/onlinestore/payment/service/PaymentScenarioHandler.java

```diff
@@ -0,0 +1,9 @@
+package com.zufar.onlinestore.payment.service;
+
+import com.stripe.model.PaymentIntent;
+
+public interface PaymentScenarioHandler {
```

Comment:
Please put all classes in the service package into new packages with correct names logically.
For example you may put all classes like `PaymentScenarioHandler`, `PaymentSuccessfulScenarioHandler`, `PaymentFailedScenarioHandler` and others into the new package `com.zufar.onlinestore.payment.service.scenario;`